src/services/__tests__/MatchService.unit.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { BallChaserQueueBuilder } from "../../../.jest/Builder";
+import { createRandomMatch } from "../MatchService";
+import { mocked } from "ts-jest/utils";
+import ActiveMatchRepository from "../../repositories/ActiveMatchRepository";
+import QueueRepository from "../../repositories/QueueRepository";
+import { Team } from "../../types/common";
+import { PlayerInQueue } from "../../repositories/QueueRepository/types";
+
+jest.mock("../../repositories/ActiveMatchRepository");
+jest.mock("../../repositories/QueueRepository");
+jest.mock("../../utils/getEnvVariable");
+
+beforeEach(async () => {
+  jest.clearAllMocks();
+});
+
+describe("Match Service tests", () => {
+  it("random teams are created", async () => {
+    const mockBallChasers = BallChaserQueueBuilder.many(6, { team: null });
+    mocked(QueueRepository.getAllBallChasersInQueue).mockImplementationOnce(() =>
+      Promise.resolve(mockBallChasers as Readonly<PlayerInQueue[]>)
+    );
+
+    let players = await createRandomMatch();
+
+    mocked(ActiveMatchRepository.addActiveMatch);
```

Comment:
Is this necessary? It doesn't seem like you're doing anything with this mock.

Reply:
Fixed.